resources/sdk/clisdkclient/extensions/restclient/tls/generate_cert.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Source - https://github.com/golang/go/blob/master/src/crypto/tls/generate_cert.go
+// or $(go env GOROOT)/src/crypto/tls/generate_cert.go
+
+// Generate a self-signed X.509 certificate for a TLS server. Outputs to
+// 'cert.pem' and 'key.pem' and will overwrite existing files.
+
+package tls
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"log"
+	"math/big"
+	"os"
+	"time"
+)
+
+var (
+	host     = "localhost"
+	validFor = 365*24*time.Hour
+	rsaBits  = 2048
+)
+
+func GenerateCerts() error {
+	var priv interface{}
+	var err error
+
+	priv, err = rsa.GenerateKey(rand.Reader, rsaBits)
```

Comment:
Should this error be checked and returned if necessary?

Reply:
Yes! It must have been lost in the refactoring